style/ember/README.md

```diff
@@ -2,6 +2,7 @@ Ember
 =====
 
 * Don't put a space between the opening handlebars braces and the variable.
+* Don't use {{partial}} helper. Use a component with only a template.
```

Comment:
I don't think there is a good reason to use `partial` over a component. The implicit context is unclear. In Rails we always say "Don't use instance variables in partials. Pass local variables to partials from view templates". I feel the same way here.